README.md

```diff
@@ -2056,6 +2056,26 @@ the current timestamps. Query cache can be enabled after the backfilling is comp
 An alternative solution is to query `/internal/resetRollupResultCache` url after backfilling is complete. This will reset
 the query cache, which could contain incomplete data cached during the backfilling.
```

Comment:
```suggestion
An alternative solution is to query [/internal/resetRollupResultCache](https://docs.victoriametrics.com/url-examples.html#internalresetRollupResultCache) handler after the backfilling is complete. This will reset the query cache, which could contain incomplete data cached during the backfilling.
```